packages/flutter/lib/src/material/app.dart

```diff
@@ -185,14 +183,11 @@ class MaterialApp extends StatefulWidget {
   ///
   /// If [home] is specified, then [routes] must not include an entry for `/`,
   /// as [home] takes its place.
-  final Widget home;
-
-  /// The primary color to use for the application in the operating system
-  /// interface.
   ///
-  /// For example, on Android this is the color used for the application in the
-  /// application switcher.
-  final Color color;
+  /// The [Navigator] is only built if routes are provided (either via [home],
+  /// [routes], [onGenerateRoute], or [onUnknownRoute]); if they are not,
+  /// [builder] must not be null.
+  final Widget home;
```

Comment:
Maybe add one more line that says the difference between using `home` vs `builder` is that home is below a Navigator and builder has no Navigator.

Reply:
done